packages/nodejs/test/registry.ts

```diff
@@ -0,0 +1,46 @@
+import { Span, SpanOptions, SpanContext, SpanData } from "../src/interfaces"
+import { RootSpan, ChildSpan } from "../src/span"
+
+export class TestRegistry {
```

Comment:
Consider giving this a more specific name about _what_ it registers (`ChildSpanRegistry`?)

Reply:
What about `SpanTestRegistry`?

Reply:
Tracking them all in one registry sounds good to me too.